src/main/java/com/zeepy/server/building/service/BuildingDealService.java

```diff
@@ -0,0 +1,87 @@
+package com.zeepy.server.building.service;
+
+import com.zeepy.server.building.domain.Building;
+import com.zeepy.server.building.domain.BuildingDeal;
+import com.zeepy.server.building.dto.BuildingDealRequestDto;
+import com.zeepy.server.building.dto.BuildingDealResponseDto;
+import com.zeepy.server.building.repository.BuildingDealRepository;
+import com.zeepy.server.building.repository.BuildingRepository;
+import com.zeepy.server.common.CustomExceptionHandler.CustomException.NoContentException;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+/**
+ * Created by Minky on 2021-05-15
+ */
+
+@Service
+@RequiredArgsConstructor(access = AccessLevel.PUBLIC)
+public class BuildingDealService {
+    private final BuildingDealRepository buildingDealRepository;
+    private final BuildingRepository buildingRepository;
+
+    // CREATE
+    @Transactional
+    @ExceptionHandler(NoContentException.class)
+    public Long create(BuildingDealRequestDto buildingDealRequestDto) {
+        Building building = buildingRepository
+                .findById(buildingDealRequestDto.getBuildingId())
+                .orElseThrow(() -> new NoContentException());
```

Comment:
```suggestion
                .orElseThrow(NoContentException::new);
```
아마 이렇게 람다식으로 가능할겁니다.